### PR TITLE
Seeds

### LINKED ISSUE #44: Polish README

### PR DESCRIPTION
**Pull request by sagemlee**
### Description
Add seeds for database mapped out on miro board.

### Type of change
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] Refactor (non-breaking change which maintains existing functionality)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -1,15 +1,105 @@
...
+phrase_12 = Phrase.create(expression: "I would like to make a deposit", image: "", tags:['Transactions'])
+phrase_13 = Phrase.create(expression: "I would like to order a new ATM/Debit card", image: "", tags:['Transactions'])
+phrase_14 = Phrase.create(expression: "I would like to open an account", image: "", tags:['Transactions'])
+phrase_15 = Phrase.create(expression: "I would like to pay my mortgage", image: "", tags:['Transactions'])
+#Bank/Goodbyes
+phrase_16 = Phrase.create(expression: "Thanks, have a great day.", image: "", tags:['Goodbyes'])
+phrase_17 = Phrase.create(expression: "Don't work too hard.", image: "", tags:['Goodbyes'])
+phrase_18 = Phrase.create(expression: "Thanks.", image: "", tags:['Goodbyes'])
+#Party/Greetings
+phrase_19 = Phrase.create(expression: "What's up y'all?", image: "", tags:['Greetings'])
```
Comment (cgaddis36) [inline db/seeds.rb:33]: this is awesome

File: db/seeds.rb [hunk 2]
```diff
@@ -1,15 +1,105 @@
...
+SectionPhrase.create(section_id: section_2.id, phrase_id: phrase_26.id)
+SectionPhrase.create(section_id: section_2.id, phrase_id: phrase_27.id)
+SectionPhrase.create(section_id: section_2.id, phrase_id: phrase_28.id)
+SectionPhrase.create(section_id: section_2.id, phrase_id: phrase_29.id)
+SectionPhrase.create(section_id: section_2.id, phrase_id: phrase_30.id)
+
+SectionPhrase.create(section_id: section_2.id, phrase_id: phrase_31.id)
+SectionPhrase.create(section_id: section_2.id, phrase_id: phrase_32.id)
+SectionPhrase.create(section_id: section_2.id, phrase_id: phrase_33.id)
+SectionPhrase.create(section_id: section_2.id, phrase_id: phrase_34.id)
```
Comment (cgaddis36) [inline db/seeds.rb:102]: thank you Sage!

File: db/seeds.rb (hunk 1 above)
Comment (EricLarson2020) [reply db/seeds.rb:33]: Great work here!!!

**Review by EricLarson2020 — APPROVED** [review]

**Review by cgaddis36 — APPROVED** [review]
Looks great! ready for merge to main

**Pull request merged**